modules/monitoring-configuring-remote-write.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="configuring_remote_write_storage_{context}"]
+= Configuring remote write storage
+
+You can configure remote write to enable Prometheus to send ingested metrics to remote systems for storage. 
+Doing so has no impact on how or for how long Prometheus stores metrics using local storage.
+
+.Prerequisites
+
+* *If you are configuring core OKD monitoring components:*
+** You have access to the cluster as a user with the `cluster-admin` role.
+** You have created the `cluster-monitoring-config` `ConfigMap` object.
+* *If you are configuring components that monitor user-defined projects:*
+** You have access to the cluster as a user with the `cluster-admin` role or as a user with the `user-workload-monitoring-config-edit` role in the `openshift-user-workload-monitoring` project.
```

Comment:
Is this a duplicate 'requirement'? How can / could this be better framed/reframed to simplify the instructions?

Reply:
These lines are copied from and have precedents in various other files, such as `modules/monitoring-configuring-a-local-persistent-volume-claim.adoc`. It would be better to stick with precedent for this PR, and create a separate enhancement request to improve the wording and content structure across all the precedents.

Reply:
@sferich888, you have a good point, but I agree with @rolfedh here for this PR. I'm leaving the wording as-is for now to stay consistent with the rest of the docs.